hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -61,6 +65,9 @@
  */
 public class AbfsClient implements Closeable {
   public static final Logger LOG = LoggerFactory.getLogger(AbfsClient.class);
+  private static final String SERVER_SIDE_ENCRYPTION_ALGORITHM = "AES256";
```

Comment:
Move these to FileSystemConfigurations.java

Reply:
Done